tests/unit/staff/views/test_users.py

```diff
@@ -588,11 +588,17 @@ def test_userlist_filter_by_invalid_org(rf, staff_area_administrator):
     assert len(response.context_data["object_list"]) == 0
 
 
-def test_userlist_filter_by_role(rf, staff_area_administrator):
-    UserFactory(roles=[OutputPublisher])
-    UserFactory(roles=[ProjectCollaborator])
-
-    request = rf.get("/?role=OutputPublisher")
+@pytest.mark.parametrize(
+    "role",
+    [
+        pytest.param(OutputPublisher, id="global"),
+        pytest.param(ProjectDeveloper, id="local"),
```

Comment:
I can see we use 'local' as an antonym to 'global' for `ProjectMembership`-based roles vs. `User`-based roles in a few places. I find this wording a little tough to intuit the meaning of and wonder if "user-based" vs "project-based" might be simpler. Or "global" vs "non-global" or "project" vs "non-project". Hmm.

I'm not suggesting this needs to change here.

Reply:
Yup, we do. And yup, "global" vs "non-global" makes sense; as does "project" vs "non-project". (We also implemented, but don't actually use, `OrgMembership`-based roles, just to 🤯.)